content/en/providers/cloud-compute/openstack/aai.md

```diff
@@ -315,12 +328,39 @@ $ cat mapping.egi.json
             }
         ]
     }
```

Comment:
```suggestion
    },
```

Reply:
Also the result of the `openstack mapping create` below it's not entirely true, but adding the actual output will result in something too verbose so I suggest to remove the output of the command